src/status_im/contexts/onboarding/events.cljs

```diff
@@ -73,7 +73,7 @@
                                       :onboarding/navigated-to-enter-seed-phrase-from-screen
                                       :screen/onboarding.new-to-status)]]
                :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
-                                 :dispatch [:onboarding/navigate-to-identifiers]}]
+                                 :dispatch [:onboarding/navigate-to-enable-notifications]}]
```

Comment:
i still not sure it's proper way to do it, because it seems like it will navigate to the new stack on top of current stack which is wrong, we need to close (remove) current stack first

Reply:
yes this is wrong, we probably messed up rebase somewhere. We decided use [init-root](https://github.com/status-im/status-mobile/pull/20503/files#r1654635905) instead

![image](https://github.com/status-im/status-mobile/assets/17097240/5d792ee2-e850-4689-b107-7c81a0bb445b)

Reply:
I think we should define root in roots file first, before calling init-root

Reply:
init-root won't work, because we need to show this screen blur on top of onboarding animation, but probably that's fine and just simpler to use init-root I agree

Reply:
Unrelated discussion, but it seems roots file is messed up. We have unused roots, not sure why they defined there
`screen/onboarding.welcome`

Reply:
@flexsurfer as is these two screens will not show over the current context `isscreen/onboarding.welcome`,` screen/onboarding.enable-notifications` since am using init-root. I think it would be okay to merge the pr first and follow it up with another pr to address.